code/datums/loadout/item_representation/_item_representation.dm

```diff
@@ -118,3 +118,32 @@
 	stack.update_weight()
 	stack.update_icon()
 	return stack
+
+/**
+ * Allow to representate an id card (/obj/item/card/id)
+ */
+/datum/item_representation/id
+	/// the access of the id
+	var/list/access = list()
+	/// the iff signal registered on the id
```

Comment:
Will this not create free clonable id cards for marines

Reply:
this is only for admins (through loadout tool). The id slot is not saved by the loadout vendors

Reply:
And even if you put an id in your bag, you can't find any in vendors. So the loadout vendor would not sell it back to you

Reply:
ok